.travis/install.sh

```diff
@@ -24,7 +24,8 @@ sudo apt-get install -qq -y \
    php-cli \
    ruby \
    rustc \
-   swi-prolog-nox
+   swi-prolog-nox \
+   scala
```

Comment:
This dependency list looks like it was (approximately) sorted alphabetically, could you place the `scala` package above the `swi-prolog-nox` one?

Reply:
Oh, yes my mistake.